ui/page/proposal/proposal_details_page.go

```diff
@@ -93,10 +101,46 @@ func (pg *proposalDetails) Handle() {
 	}
 
 	if pg.vote.Button.Clicked() {
-		newVoteModal(pg.Load).Show()
+		newVoteModal(pg.Load, pg.proposal).Show()
+	}
+
+	for pg.viewInPoliteiaBtn.Clicked() {
+		components.GoToURL("https://proposals.decred.org/record/" + pg.proposal.Token)
 	}
 }
 
+func (pg *proposalDetails) listenForSyncNotifications() {
+	go func() {
+		for {
+			var notification interface{}
+
+			select {
+			case notification = <-pg.Receiver.NotificationsUpdate:
+			default:
+				if components.ContextDone(pg.ctx) {
+					return
+				}
+			}
+
+			switch n := notification.(type) {
+			case wallet.Proposal:
+				if n.ProposalStatus == wallet.Synced {
+					proposal, err := pg.WL.MultiWallet.Politeia.GetProposalRaw(pg.proposal.Token)
+					if err == nil {
+						pg.proposal = proposal
+						pg.RefreshWindow()
+					}
+				}
+			}
+		}
+	}()
+}
+func (pg *proposalDetails) OnClose() {
+	pg.ctxCancel()
+}
+
+// - Layout
```

Comment:
Remove.

Reply:
This is used to divide the go file into the frontend and backend sections of a page.